### ISSUE TITLE
Crash w/ World of Wonder

### ISSUE THREAD

**Issue by WenXin20** [ShetiPhian/Localizations-and-Issues#277]
`Minecraft v1.15.2`
`Forge v31.2.45`
`Terraqueous v1.6.4`
[`World of Wonder v1.1.4`](https://www.curseforge.com/minecraft/mc-mods/world-of-wonder)

Hello, I got a strange crash with World of Wonder while loading a new world on my server. The biome the other mod adds is named Dandelion Fields.
[Crash Report](https://gist.github.com/WenXin20/0c56fa6cbacd34971af6ef06642da996)

**Comment by ShetiPhian** [comment]
Interesting....
So I can stop this crash with a simple check, but this issue is bigger then a simple compatibility issue.

My BlockLeavesBase is mostly a copy of the vanilla LeavesBlock (only using my own to work around hard coded logic)
In both classes the getDistance method assumes any block in the leaves tag has the distance property.

This is not always the case, the World of Wonder block is an example of this.

In short:
Any leaf block that extends LeavesBlock will crash if it checks a neighbor block that is registered to the leaves tag but doesn't have the distance property.

**Comment by WenXin20** [comment]
So I guess this is happening because I added the Dandelion Fluff to the leaves tag?

**Comment by ShetiPhian** [comment]
Kinda, but it does point out an error in the base game itself.
Its completely valid to have leaves not use the distance property, but the default leaves block expects it to be there.

**Comment by WenXin20** [comment]
Ah, I see

**Comment by ShetiPhian** [comment]
New build is now on Curse.
https://www.curseforge.com/minecraft/mc-mods/terraqueous/files/3106992

**Comment by WenXin20** [comment]
Alright, thanks!
